lib/java/com/google/android/material/chip/ChipDrawable.java

```diff
@@ -183,6 +183,7 @@
   @Nullable private ColorStateList chipIconTint;
   private float chipIconSize;
   private boolean hasChipIconTint;
+  private boolean chipIconUseDrawableSize;
```

Comment:
this variable name sounds awkward and a little mis-leading since chip will not always use the drawable's height. What about `useChipIconDrawableSize`?

Reply:
Changed to `useChipIconDrawableSize`, it is a better name.